Cargo.toml

```diff
@@ -28,6 +28,7 @@ alloy-transport = { git = "https://github.com/alloy-rs/alloy", rev = "a4bb5f0be3
 alloy-transport-http = { git = "https://github.com/alloy-rs/alloy", rev = "a4bb5f0be3eec5c8679bdab93c1482df38ba8509" }
 reqwest = "0.12.4"
 tokio = { version = "1.38.0", features = ["rt-multi-thread"] }
+alloy-chains = { version = "0.1.22", features = ["serde"] }
```

Comment:
Let's put this above `alloy-primitives`. This section is for the RPC storage dependencies.